### ISSUE TITLE
Protege/Patron Tinder

### ISSUE THREAD

**Issue by Zironic** [Arx-Game/arxcode#79]
I think it would be relatively easy to make a system where you can fill in your characters primary focus out of a list of of choices and their primary timezones and then give them a list of people looking for a protege/sponsor matching those criteria.

That would give people a pretty easy way to find out who's available and then contact them for RP. It wouldn't need to be very deep since actual character chemistry is something you only find out during the scene.

**Comment by Zironic** [comment]
A similar system could be used for finding people interested in the same plot as you.
"You've marked an interest in clue X, Y and Z.  These people might share your interest:"

**Comment by TehomCD** [comment]
Yeah, I was just thinking that it'd probably fit for anything of 'find X person in same timezone for Y'. I'm unsure if there's enough overlap with petitions where we'd wanna look at usability improvements there to make it work for it, or if it's something that we want to handle separately. Petitions are typically aimed at orgs, so I'm lean towards thinking they're distinct.

**Comment by rozzingit** [comment]
I think you _could_ use petitions for this if you make some heavy revisions to maybe add capability for either tagging or categorizing them. Public petitions (not org ones) seem a fairly common host for queries about patronage, _but_ I think that petitions have been a lot more successful for org usage than they have been for open/public usage.

As far as timezones go, they can be useful BUT just saying what your timezone is isn't necessarily an indicator of when you can _play_. So playable hours is the more useful search.

**Comment by Zironic** [comment]
While this might possibly end up under the petition set of systems, I don't think the petition system itself is a good fit for it because you really don't want to make people manually go through a bunch of effectively classifieds trying to figure out ones are in some way relevant. Petitions themselves do need a bunch of improvements someone ought to write a ticket for.

What you want is something that just lets them state some simple preferences and it'll start spitting out names to get in contact with.

**Comment by rozzingit** [comment]
I also wouldn't underestimate the number of people who might legitimately not really have preferences and will _want_ to kind of just flip through a list of possibilities.

Oh! But that does remind me: auto-filter by the patronage SR rules!

**Comment by Zironic** [comment]
I think once I'm done with all the UX improvements I'll take a stab at making this matching system. Having no preferences would be fine, you'd just get introduced to all the people. Automatically filtering by the patronage rules would make a lot of sense for anyone looking for a patron/protege.

The main use cases I'm seeing right now are people looking for Patrons/Proteges as well as looking for a plot buddy.
Another use case I could see is marriage tinder, I'm seeing a lot of people with 'marriage' in their rp hook field and they might appreciate an easy way to see the other noble singles.
